bucket/p4v.json

```
@@ -0,0 +1,44 @@
+{
+    "version": "2021.4/2227050",
+    "description": "Helix Visual Client (P4V) is a desktop app that provides access to versioned files in Helix Core through a graphical interface.",
```

```suggestion
    "description": "Provides access to versioned files in Helix Core through a graphical interface.",
```
We tend to keep description shorter, so that we can fill for information inside it.